pages/common/aria2c.md

```diff
@@ -12,7 +12,11 @@
 
 `aria2c -o {{filename}} {{url}}`
 
-- Download from multiple sources:
+- Download parallel from multiple sources each URI pointing to a different file:
```

Comment:
I think we can simplify this example description:
```suggestion
- Download multiple files in parallel:
```

Reply:
I partially agree with you, I had given the longer description so that one can clear understand what happens with and without `--force-sequential` option, which is written just below it. 

Keeping in mind that most people are habituated with using `wget` the above reason makes more sense.

Reply:
From my personal experience using `aria2c` for 1 year and more. I think keeping the longer variant will be better.

Reply:
@gurditsbedi I'm not sure why longer description is better, because that's all implied by the short description.  No one would think that it's downloading the same file several times in parallel 😄 — on the contrary, everyone would think that `aria2c` would download each file pointed to by the URLs at the same time.  Is this incorrect?

Reply:
I got what ya saying @bl-ue , the file will be downloaded parallel, but there is limit to parallel downloads by default which is `5` and is controlled `--max-concurrent-downloads` option. 

I don't have any issue with the shorter description.

Should I add a tldr for `--max-concurrent-downloads` also?

Reply:
Sure! That would be great.

Reply:
I have added `--max-concurrent-downloads`.

Reply:
@gurditsbedi The pages can't have more than 8 examples.

Reply:
Done. :)